src/Alert.php

```diff
@@ -27,23 +27,45 @@
  */
 final class Alert extends Widget
 {
+    private const TYPE_PRIMARY = 'alert-primary';
+    private const TYPE_SECONDARY = 'alert-secondary';
+    private const TYPE_SUCCESS = 'alert-success';
+    private const TYPE_DANGER = 'alert-danger';
+    private const TYPE_WARNING = 'alert-warning';
+    private const TYPE_INFO = 'alert-info';
+    private const TYPE_LIGHT = 'alert-light';
+    private const TYPE_DARK = 'alert-dark';
```

Comment:
This constants uses once, suggest remove constants and set classes directly in methods.